### PR TITLE
[tests-only][full-ci] Added WebUI tests for checking version author

### LINKED ISSUE #39504: [tests-only] Add acceptance tests for version metadata
See PR  #39126 

oC10 core is being enhanced to remember the author of each version of a file.

Add acceptance tests to cover this new feature:
- API tests to check that the author of each version can be returned in API requests
- webUI tests to check that the author is displayed against each version in the version history
- also have test cases to check what happens when the feature is enabled and there are existing files that are then edited after the feature is enabled.

Here is an example screenshot of the feature:

edits-by-group-members

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
added webui acceptance tests for versions author

## Related Issue
Closes https://github.com/owncloud/core/issues/39504

## Motivation and Context
Test coverage for new feature

## How Has This Been Tested?
- test environment: local

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

File: tests/acceptance/features/webUIFiles/versions.feature [hunk 1]
```diff
@@ -90,3 +90,63 @@ Feature: Versions of a file
...
+    And user "Brian" has uploaded file with content "Brian new lorem content" to "/randomfile.txt"
+    And user "Carol" has logged in using the webUI
+    And the user has browsed to the files page
+    When the user browses directly to display the "versions" details of file "randomfile.txt" in folder "/"
+    Then the authors of the versions of file "randomfile.txt" should be:
+      | index | author |
+      | 1     | Brian  |
+      | 2     | Alice  |
+      | 3     | Alice  |
+
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFiles/versions.feature:127]: Shouldn't `Brian` have 2 versions as he uploaded twice? Or is that not how this works?

File: tests/acceptance/features/webUIFiles/versions.feature [hunk 2]
```diff
@@ -90,3 +90,63 @@ Feature: Versions of a file
...
+  Scenario: sharee can see the versions' respective author after version restore
+    Given the administrator has enabled the file version storage feature
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Carol" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "some content" to "/randomfile.txt"
+    And user "Alice" has shared file "randomfile.txt" with user "Brian"
+    And user "Alice" has shared file "randomfile.txt" with user "Carol"
+    And user "Brian" has uploaded file with content "Brian lorem content" to "/randomfile.txt"
+    And user "Alice" has uploaded file with content "new lorem content" to "/randomfile.txt"
+    And user "Carol" has uploaded file with content "Brian new lorem content" to "/randomfile.txt"
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFiles/versions.feature:138]:
```suggestion
    And user "Carol" has uploaded file with content "Carol new lorem content" to "/randomfile.txt"
```

File: tests/acceptance/features/webUIFiles/versions.feature (hunk 1 above)
Comment (SwikritiT) [reply tests/acceptance/features/webUIFiles/versions.feature:127]: Oh I got it

Comment (phil-davis) [reply tests/acceptance/features/webUIFiles/versions.feature:127]: Note to future readers: Brian did 2 uploads, so Brian is the author of the current file (not in the versions list) and of the latest version (index 1)

**Review by phil-davis — APPROVED** [review]

**Pull request merged**